documentation/messages/user.registration.transactional.md

```diff
@@ -94,10 +94,7 @@ Accessed by connecting to RabbitMQ server and sending message in the following f
   /* Required: user_id or email or mobile. */
   mobile: Integer,
 
-  /* Optional. A string to tag the message with that can be referenced in Mandrill. Example: `drupal_user_register`. */
-  registration_source: String,
-
-  /* Optional. Used to determine the origin of the request as a site in multisite environment. Default: `US`. */
+  /* Optional. Client application identifier. Example: `Phoenix-US`. */
```

Comment:
Created a label: "Future Issue" (https://github.com/DoSomething/quicksilver-api/labels/Future%20Issue) in the hopes of not forgetting this point